### PR TITLE
Enable theming and add docs

### PR DESCRIPTION
**Pull request by ksiabani**

### CODE REVIEW

File: .storybook/theme.js [hunk 1]
```diff
@@ -5,5 +5,5 @@ export default create({
   fontBase: '"M PLUS Rounded 1c", sans-serif',
   brandTitle: 'Matte',
   brandImage: 'images/matte-logo-storybook.png',
-  brandUrl: '?path=/docs/*',
-})
+  brandUrl: '?path=/story/design-system-intro--page',
```
Comment (momegas) [inline .storybook/theme.js:8]: Is the double - a typo? -> `intro--page`

File: src/docs/intro.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+<Meta title="User Interface/Intro" />
+
+# Matte Design System
+
+###### MATTE stands for MATerial Typography Experience. It's build on top of [Material UI](https://material-ui.com/) and other great React libraries. It is supposed to be as good as [Material Design](https://material.io/design), but not as shiny (hence the matte appearence).
+
+TODO: Why Matte? Different, prototyping and dashboards
+TODO: Quick start guide for dashboard
+TODO: Matte vs MUI and how to use both
```
Comment (momegas) [inline src/docs/intro.stories.mdx:11]: Should these TODOs be here? Visible?

File: .storybook/theme.js (hunk 1 above)
Comment (ksiabani) [reply .storybook/theme.js:8]: It's a storybook thing

File: src/docs/intro.stories.mdx (hunk 2 above)
Comment (ksiabani) [reply src/docs/intro.stories.mdx]: Removed

File: .storybook/theme.js (hunk 1 above)
Comment (momegas) [reply .storybook/theme.js:8]: great then

**Pull request merged**